crates/loader/tests/ui/valid-manifest.lock

```diff
@@ -24,7 +24,7 @@
       "trigger_config": {
         "component": "four-lights",
         "executor": {
-          "type": "spin"
+          "type": "http"
```

Comment:
Hmm actually I might need to hate this if its going to be backward compat problem.

Reply:
I guess this does mean that Spin 1.x can't read locked apps from Spin 2.x but I think that's fine?

Reply:
Oh yeah, I guess the alias makes it forward compatible. :+1: